.drone.jsonnet

```diff
@@ -0,0 +1,191 @@
+  commands = null,
+  environment = null,
+  failure = false,
+  detach = false,
+  privileged = false,
+  volumes = [],
+  when = null
+) = {
+  name: name,
+  image: image,
+  [if failure then 'failure']: failure,
+  [if detach then 'detach']: detach,
+  [if privileged then 'privileged']: detach,
+  [if settings != null then 'settings']: settings,
+  [if depends_on != [] then 'depends_on']: depends_on,
+  [if commands != [] then 'commands']: commands,
+  [if environment != null then 'environment']: environment,
+  [if volumes != [] then 'volumes']: volumes,
+  [if when != null then 'when']: when,
+};
+
+[
+  pipeline(
+    kind = "pipeline",
+    name = "checks (node:10)",
+    steps = [
+      step(
+        name = "build information",
+        image = "node:10",
+        commands = [
+          "node --version",
+          "npm --version",
+          "git --version",
+        ],
+      ),
+      step(
+        name = "install",
+        image = "node:10",
+        commands = [
+          "npm install",
+        ],
+      ),
+      step(
+        name = "lint",
+        image = "node:10",
+        commands = [
+          "npm run lint",
+        ],
+      ),
+    ],
+  ),
+  pipeline(
+    kind = "pipeline",
+    name = "unit tests (node:6)",
+    steps = [
+      step(
+        name = "build information",
+        image = "node:6",
+        commands = [
+          "node --version",
+          "npm --version",
+          "git --version",
+        ],
+      ),
+      step(
+        name = "install",
+        image = "node:6",
+        commands = [
+          "npm install",
+        ],
+      ),
+      step(
+        name = "unit-test",
+        image = "node:6",
+        depends_on = [
+          "install",
+        ],
+        commands = [
+          "npm run test",
+        ],
+        environment = {
+          MONGO_URI: "mongodb://mongo:27017/key-recovery-service-test",
+        },
+      ),
+    ],
+    trigger = {
+      branch: {
+        exclude: [
+          "prod/production",
+        ],
+      },
+    },
+    services = [
+      {
+        name: "mongo",
+        image: "mongo:3.4",
+      },
+    ]
+  ),
+  pipeline(
+    kind = "pipeline",
+    name = "unit tests (node:10)",
+    steps = [
+      step(
+        name = "build information",
+        image = "node:10",
+        commands = [
+          "node --version",
+          "npm --version",
+          "git --version",
+        ],
+      ),
+      step(
+        name = "install",
+        image = "node:10",
+        commands = [
+          "npm install",
+        ],
+      ),
+      step(
+        name = "unit-test",
+        image = "node:10",
+        depends_on = [
+          "install",
+        ],
+        commands = [
+          "npm run test",
+        ],
+        environment = {
+          MONGO_URI: "mongodb://mongo:27017/key-recovery-service-test",
+        },
+      ),
+      step(
+        name = "audit",
+        image = "node:10",
+        failure = true,
```

Comment:
Does this mean fail on error or ignore error?

Reply:
It is close to "ignore error" and do not stop the run or fail entire pipeline.